InstallingDrivers.md

```diff
@@ -120,3 +120,78 @@ _Note: Only Skylake, Kaby Lake, and Coffee Lake offer full Vulkan support. Broad
 To install support for Vulkan API (will be functional only if you have a [Vulkan capable GPU](https://en.wikipedia.org/wiki/Vulkan_(API)#Compatibility) and driver), execute following command:
 
     sudo dnf install vulkan-loader vulkan-loader.i686
+
+    
+## openSUSE Tumbleweed/Leap
```

Comment:
We don't support current latest Leap, so it should be removed from here.

Reply:
Which version of Python 3 is required at least? Leap 15.2 has 3.6.10 available if that's is the issue I'll remove Leap here

As for Lutris wine builds glibc is available for Leap 15.2 in version: 2.26 I remember having issues with that one on Leap 15.1 since your newer wine builds need at least 2.27 if I am not mistaken? If so yes I'll remove Leap here.

Reply:
Yeah the issue is with glibc, 2.26 is too old.
And yeah do mention that.